demo/src/app/default/default.component.html

```diff
@@ -3,14 +3,22 @@
     <div class="pull-lg-left">
       <h1 class="display-3">ng-bootstrap</h1>
       <hr class="m-y-2 header">
-      <p>Angular 2, powered by Bootstrap 4 written by the angular-ui Team</p>
+      <p class="lead">Angular 2, powered by Bootstrap 4 written by the angular-ui Team</p>
+      <p class="version">Currently v-alpha.1</p>
```

Comment:
This would be pain in a neck to maintain by hand. We need to automate it if we want to display version (and we want in the long run). Please remove for now.